### PR TITLE
Add docs for Docker images

### PR DESCRIPTION
**Pull request by jarpy**
- Add Docker image info to the Getting Started pages.
- Add/extend dedicated Docker page for each (Unix) Beat.
- Refactor some duplicated fragments into includes with attributes.
- Remove some whitespaces (sorry for the noisy diff, my editor just did this stuff).

### CODE REVIEW

File: libbeat/docs/shared-docker.asciidoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+docker pull {dockerimage}
+------------------------------------------------
+
+endif::[]
+
+The base image is https://hub.docker.com/_/centos/[Centos 7] and the source
+code can be found on
+https://github.com/elastic/beats-docker/tree/{doc-branch}[GitHub].
+
+=== Configuring {beatname_uc} on Docker
```
Comment (jarpy) [inline libbeat/docs/shared-docker.asciidoc]: Thanks! I added the `[float]` tags; it looks much better now.

I'm glad I didn't cross a line with the refactors. I always get nervous pushing patches for new projects in new languages. :)

File: metricbeat/docs/running-on-docker.asciidoc [hunk 2]
```diff
@@ -1,45 +1,25 @@
...
-endif::[]
+include::../../libbeat/docs/shared-docker.asciidoc[]
 
+[float]
+[[monitoring-host]]
+=== Monitoring the Host Machine
 When executing Metricbeat in a container, there are some important
 things to be aware of if you want to monitor the host machine or other
 containers. Let's walk-through some examples using Docker as our container
 orchestration tool.
```
Comment (dliappis) [inline metricbeat/docs/running-on-docker.asciidoc]: A few clarifications or warnings could be provided for Docker on Windows + Mac users where the bind mounts won't work as expected.

File: packetbeat/docs/running-on-docker.asciidoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+docker run --cap-add=NET_ADMIN {dockerimage}
+----
+
+=== Capturing Traffic from the Host System
+
+By default, Docker networking will connect the Packetbeat container to an
+isolated virtual network, with a limited view of network traffic. You may wish
+to connect the container directly to the host network in order to see traffic
+destined for, and originating from, the host system. With +docker run+, this can
+be achieved by specifying +--network=host+.
```
Comment (dliappis) [inline packetbeat/docs/running-on-docker.asciidoc]: Same comment here, for users on Docker for Mac+Windows have a VM layer that won't let them sniff things happening on the real host OS, unless settings have been tweaker.

**Comment by jarpy** [comment]
Thanks, @dliappis. Added your suggestions.

**Comment by jarpy** [comment]
OK to merge this now, @ruflin?

**Comment by ruflin** [comment]
Happy to merge if @dliappis gives a 👍

**Comment by ruflin** [comment]
jenkins, retest it

**Comment by dliappis** [comment]
LGTM from me as well

**Comment by ruflin** [comment]
@dedemorton Can you take care of the backporting in case we backport this?

**Comment by dedemorton** [comment]
@ruflin Sure. I just need to backport this to 5.4 right?

**Comment by jarpy** [comment]
@dedemorton: yes, thanks. The images in question existed in "alpha" state earlier, but I think it's best to call 5.4 the release version, and thus only backport that far.

**Pull request merged**